### PR TITLE
Move natures into their own file

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: sim/dex.ts [hunk 1]
```diff
@@ -813,19 +781,29 @@ export class ModdedDex {
...
+			nature = new Data.Nature(natureData);
+			if (nature.gen > this.gen) nature.isNonstandard = 'Future';
+		} else {
+			nature = new Data.Nature({id, name, exists: false});
+		}
 
+		if (nature.exists) {
+			if (!!nature.plus !== !!nature.minus) {
+				throw new Error(`Nature with a '${nature.plus ? 'plus' : 'minus'}' property but no '${nature.plus ? 'minus' : 'plus'}' property: ${nature.name}`);
+			}
```
Comment (Zarel) [inline sim/dex.ts:804]: This belongs in `test/data` I think. The engine itself doesn't require both, does it?

Comment (Zarel) [reply sim/dex.ts:804]: Either way, I'd rather test this once for free than every time someone uses `Dex.getNature`.

**Comment by Zarel** [comment]
I believe everything here is good. I'll handle the tests in a follow-up commit.

File: sim/dex.ts (hunk 1 above)
Comment (KrisXV) [reply sim/dex.ts:804]: oh mb

**Pull request merged**